### DISCUSSION TITLE
Optional delay for swipe gesture

### DISCUSSION THREAD

**Discussion by JacksonWrath** [riley-martin/gestures#19] [Show and tell]
Hi! I started using this to enable "three finger drag" as has been discussed before, but I found myself wishing I could lift and continue the drag. So I implemented a way to configure a delay for the end command of the swipe gesture.

Thoughts on this? Want a PR / changes and then a PR?

https://github.com/JacksonWrath/gestures/tree/continue-delay

**Comment by riley-martin** [comment]
This looks like a nice feature to have, thanks for the suggestion. From a quick look the code seems good but feel free to make a PR and I'll review it more thoroughly and suggest changes there.
